pages/contact.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
+import NavbarBootstrap from '../components/NavbarBootstrap';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
```

Comment:
this is really good to import these components in this way 💯 

we could also do something like this:

```
import {
    Row,
    Col,
    Button,
    Card
} from 'react-bootstrap';
```

but the way we are doing imports is much better because doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client

Reply:
Oh that's neat! Thanks for letting me know 😊